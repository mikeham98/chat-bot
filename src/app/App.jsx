import React from 'react';
import ThemeContainer from "./containers/Theme/ThemeContainer";
import ChatBotLayout from "./layout/ChatBotLayout";
import ConversationPanelLayout from "./layout/ConversationPanelLayout";
import styles from '../themes/components/layout.scss';

export default class App extends React.Component {
    render() {
        return (
            <ThemeContainer>
                <div className={styles.chatBotContainer}>
                    <div className={styles.chatBotWrapper}>
                        <ConversationPanelLayout/>
                        <div className={styles.messageSection}>
                            <ChatBotLayout/>
                        </div>
                    </div>
                </div>
            </ThemeContainer>
        );
    }
}