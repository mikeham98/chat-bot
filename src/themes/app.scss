body {
  margin: 0;
  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Noto Sans, Ubuntu, Droid Sans, Helvetica Neue, sans-serif;
  color: #0e0a1f;
}

input:focus, button:focus, textarea:focus {
  outline: none;
}

svg {
  transition: fill ease-in-out 0.2s;
}