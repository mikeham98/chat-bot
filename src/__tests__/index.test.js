import React from 'react';
import ReactDOM from 'react-dom';
import '../app/index';

jest.mock('../app/store', () => ({
    common: () => {},
}));
jest.mock('react-dom', () => ({
    render: jest.fn()
}));

describe('index', () => {
    it('should return a Provider with App', () => {
        expect(ReactDOM.render).toMatchSnapshot();
    });
});