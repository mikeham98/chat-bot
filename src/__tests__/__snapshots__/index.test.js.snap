// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should return a Provider with App 1`] = `
[MockFunction] {
  "calls": Array [
    Array [
      <Provider
        store={
          Object {
            "common": [Function],
          }
        }
      >
        <App />
      </Provider>,
      null,
    ],
  ],
  "results": Array [
    Object {
      "type": "return",
      "value": undefined,
    },
  ],
}
`;
